index.html

```diff
@@ -22,47 +23,51 @@ <h1>Daniel Eicher</h1>
   </header>
   <figure>
     <img src="https://source.unsplash.com/random/600x300" alt="">
-    <figcaption>This is an example image</figcaption>
+    <figcaption>Here will be be a picture of mysfelf</figcaption>
```

Comment:
here is a little typo in the word "myself" ;)

Reply:
thx